docs/site/Model.md

```diff
@@ -851,6 +851,27 @@ class TestModel {
 }
 ```
 
+To define a nested array property, you must provide the `jsonSchema` field to
+describe the sub-array property. For example:
+
+```ts
+@model()
+class TestModel {
+  // alternatively use @property.array('array')
+  @property.array(Array, {
+    jsonSchema: {
+      type: 'array',
+      items: {type: 'string'},
```

Comment:
I have another question of the design. Do we need to specify these two fields? if so, why do we need `type: 'array'` as we already use the decorator?

Reply:
@agnes512 the `jsonSchema` here is for the sub array property, 
e.g. For `Array<Array<string>>`, the json schema describes `Array<string>`

If the current doc confuses you:

> To define a nested array property, you must provide the `jsonSchema` field to describe the sub-array property.

Any suggestion on how to make it more clear?

Reply:
Ah I see, then I am good with it 👍